crates/bevy_ui/src/layout/debug.rs

```diff
@@ -12,22 +12,31 @@ pub fn print_ui_layout_tree(ui_surface: &UiSurface) {
     let taffy_to_entity: HashMap<NodeId, Entity> = ui_surface
         .entity_to_taffy
         .iter()
-        .map(|(entity, node)| (*node, *entity))
+        .map(|(&ui_entity, &taffy_node)| (taffy_node, ui_entity))
         .collect();
-    for (&entity, roots) in &ui_surface.camera_roots {
-        let mut out = String::new();
-        for root in roots {
+    for (&camera_entity, root_node_set) in ui_surface.camera_root_nodes.iter() {
+        bevy_utils::tracing::info!("Layout tree for camera entity: {camera_entity}");
```

Comment:
```suggestion
        tracing::info!("Layout tree for camera entity: {camera_entity}");
```
(You will need to update the PR with the latest changes from the base branch, as your branch doesn't import `tracing`)